docset/winserver2025-ps/smbshare/Reset-SmbServerConfiguration.md

```diff
@@ -143,6 +142,74 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -AuditClientCertificateAccess
+
+Enables auditing of the clients certificate access attempts on the server. Events are logged when a
+client attempts to access a certificate store on the server.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -AuditClientDoesNotSupportEncryption
+
+Enables auditing of the clients attempts to connect without encryption to the server. The server
+logs an audit event when a client attempts to connect to the server without using encryption.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -AuditClientDoesNotSupportSigning
+
+Enables auditing of the clients attempts to connect without signing to the server. The server logs
+an audit event when a client attempts to connect to the server without using message signing.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -AuditInsecureGuestLogon
+
+Specifies whether to audit insecure guest logon attempts. When enabled insecure guest logons will
```

Comment:
change to the audit events will appear in the windows event logs (event viewer is just one log reading tool of many)